doc/content/getting-started/ttn/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "The Things Network"
+description: ""
+distributions: "The Things Network"
+---
+
+{{< figure src="logo_TTN.svg" class="float plain" width="200px" >}}
+
+The Things Network is a global community building an open, crowdsourced, decentralized LoRaWAN network.
+
+The Things Network runs {{% tts %}} Community, and is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
```

Comment:
```suggestion
The Things Network runs a {{% tts %}} Community Edition, which is a crowdsourced, open and decentralized LoRaWAN network. This network is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
```

Reply:
So are we using Edition only for Community Edition, and not for Enterprise and Open Source?

Reply:
Good one. cc @laurensslats

Reply:
I would personally drop Edition. But I'm also very much in favor to stop changing the branding all the time.

Reply:
also in favor of dropping edition